tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -250,7 +172,7 @@ trait WskWebActionsTests
                 response.statusCode shouldBe 200
                 response.header("Access-Control-Allow-Origin") shouldBe "*"
                 response.header("Access-Control-Allow-Methods") shouldBe "OPTIONS, GET, DELETE, POST, PUT, HEAD, PATCH"
-                response.header("Access-Control-Allow-Headers") shouldBe "Authorization, Content-Type"
+                response.header("Access-Control-Allow-Headers") shouldBe "Authorization, Content-Type, User-Agent"
```

Comment:
Maybe I am confused, but I thought this user-agent header was for our own tracking data, not any thing that is exposed to the user for customization. Would enforcing this for web actions potentially hinder users from implementing their own user-agent header? and/or also force them to add a user-agent to all their web action calls?